jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -381,6 +393,13 @@ def state_attr_for_line_visible(state):
         self.stretch_vmax = PlotOptionsSyncState(self, self.viewer, self.layer, 'v_max',
                                                  'stretch_vmax_value', 'stretch_vmax_sync',
                                                  state_filter=is_image)
+
+        self.stretch_curve_visible = PlotOptionsSyncState(self, self.viewer, self.layer,

```

Comment:
do we want to add this to the user API (and then API docs)?

Reply:
Done.